### ISSUE TITLE
Array double arrow not aligned correctly

### ISSUE THREAD

**Issue by JimmyAppelt** [squizlabs/PHP_CodeSniffer#2114]
Hi there,

Got this warning when array keys are a bit longer:

### All ok

![phpstorm64_2018-08-08_01-57-16](https://user-images.githubusercontent.com/5420393/43808672-63beb6e4-9aae-11e8-816f-b1047d0b02ae.png)

### Making key's a bit longer (prepending test_)

![phpstorm64_2018-08-08_01-56-58](https://user-images.githubusercontent.com/5420393/43808680-6f6570e6-9aae-11e8-8aa3-aae3d5ca304b.png)
![phpstorm64_2018-08-08_01-56-49](https://user-images.githubusercontent.com/5420393/43808695-7d8f62d0-9aae-11e8-99c4-40be6041008d.png)

**Comment by gsherwood** [comment]
I can't replicate the errors you are seeing. The only errors I get from that Squiz.Arrays.ArrayDeclaration sniff related to the position of the key and the closing brace:
```
FILE: temp.php
----------------------------------------------------------------------------------------------
FOUND 10 ERRORS AFFECTING 10 LINES
----------------------------------------------------------------------------------------------
  5 | ERROR | [x] Array key not aligned correctly; expected 11 spaces but found 4
  6 | ERROR | [x] Array key not aligned correctly; expected 11 spaces but found 4
  7 | ERROR | [x] Array key not aligned correctly; expected 11 spaces but found 4
  8 | ERROR | [x] Array key not aligned correctly; expected 11 spaces but found 4
  9 | ERROR | [x] Closing parenthesis not aligned correctly; expected 10 space(s) but found 0
 13 | ERROR | [x] Array key not aligned correctly; expected 11 spaces but found 4
 14 | ERROR | [x] Array key not aligned correctly; expected 11 spaces but found 4
 15 | ERROR | [x] Array key not aligned correctly; expected 11 spaces but found 4
 16 | ERROR | [x] Array key not aligned correctly; expected 11 spaces but found 4
 17 | ERROR | [x] Closing parenthesis not aligned correctly; expected 10 space(s) but found 0
----------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 10 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------------------------------
```
Are you using a custom standard, ruleset, or sniff?

**Comment by JimmyAppelt** [comment]
@gsherwood forgive me 🙂 I'm using Wordpress Standard, I'm supposed to report/log it there.
